### PR TITLE
Removed dead code about Aviary in FileManager

### LINKED ISSUE #22002: Dead JS dependency in admin-dev/filemanager/dialog.php
#### Is your feature request related to a problem?

The very old code `admin-dev/filemanager/dialog.php` has an option that rely on 2 JS scripts that are no longer there.

https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/filemanager/dialog.php#L216

```
<?php
        if ($aviary_active) {
            if (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off' || $_SERVER['SERVER_PORT'] == 443) {
                ?>
				<script type="text/javascript" src="https://dme0ih8comzn4.cloudfront.net/js/feather.js"></script>
			<?php
            } else {
                ?>
				<script type="text/javascript" src="http://feather.aviary.com/js/feather.js "></script>
			<?php

            }
        }
    ?>
```

https://dme0ih8comzn4.cloudfront.net/js/feather.js => dead
http://feather.aviary.com/js/feather.js => dead

So this option cannot work as the source code is long gone.

#### Describe the solution you'd like

Remove this option (or restore it using a good source)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Removed dead code about Aviary in FileManager
| Type?             | improvement
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #22002
| How to test?      | The filemanage runs well.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27751)

### CODE REVIEW

File: admin-dev/filemanager/dialog.php [hunk 1]
```diff
@@ -212,21 +212,6 @@
...
 		<script type="text/javascript" src="js/jquery.queryloader2.min.js"></script>
-		<?php
-        if ($aviary_active) {
-            if (!empty($_SERVER['HTTPS']) && $_SERVER['HTTPS'] !== 'off' || $_SERVER['SERVER_PORT'] == 443) {
-                ?>
-				<script type="text/javascript" src="https://dme0ih8comzn4.cloudfront.net/js/feather.js"></script>
-			<?php
-            } else {
-                ?>
-				<script type="text/javascript" src="http://feather.aviary.com/js/feather.js "></script>
```
Comment (matks) [inline admin-dev/filemanager/dialog.php:223]: A scar of the past 😄

**Review by matks — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Review by marwachelly — APPROVED** [review]
Hello,

PR is tested. I tried to sweep the majority of pages FO and BO and I did not detect any regression related to the PR. Also I executed all automatic tests and I had green result. So PR is QA approved.

Thank you,

**Comment by Progi1984** [comment]
Thanks @marwachelly

**Comment by matks** [comment]
I add label "Needs documentation". Code removal needs to be documented https://devdocs.prestashop.com/8/modules/core-updates/8.0/

**Pull request merged**